packages/cw-orch-networks/src/networks/mod.rs

```diff
@@ -11,11 +11,13 @@ pub mod neutron;
 pub mod osmosis;
 pub mod sei;
 pub mod terra;
+pub mod wasm;
 
 pub use crate::chain_info::{ChainInfo, ChainKind, NetworkInfo};
 pub use archway::{ARCHWAY_1, CONSTANTINE_3};
 pub use injective::{INJECTIVE_1, INJECTIVE_888};
 pub use juno::{JUNO_1, LOCAL_JUNO, UNI_6};
+pub use wasm::{LOCAL_WASM};
```

Comment:
Did not add to the `parse_network` function because the id conflicts with `LOCAL_JUNO`

Reply:
Sure !